lib/JiraClient.ts

```diff
@@ -74,20 +81,20 @@ export class JiraClient {
     };
   };
 
-  private async sendJiraGet(endpoint: string): Promise<any> {
-    return this.sendJiraRequest("GET", endpoint);
+  private async sendRestGet(endpoint: string): Promise<any> {
```

Comment:
The renames and refactorings here are to make room for `sendGraphQL(query: any)` that will call `sendRequest`